sdk/azidentity/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Breaking Changes
 
+* Instances of `ChainedTokenCredential` will now skip looping through the list of source credentials and re-use the first successful credential on subsequent calls to `GetToken`.
```

Comment:
Worth mentioning `DefaultAzureCredential` here--this PR makes the same change to it. The difference is that the behavior is configurable for `ChainedTokenCredential`.

Reply:
I’m assuming you meant to add `RetrySources` to the `DefaultAzureCredential`, is that correct? I added it here: https://github.com/Azure/azure-sdk-for-go/pull/16392/commits/e2d1201e28ba824b2b640a5eda0b1be1c5120fbc

Reply:
What I meant is that this entry should also name `DefaultAzureCredential` because its default behavior is changing in the same way. `DefaultAzureCredential` uses `ChainedTokenCredential`, so changing the latter's default behavior changes the former's as well. No problem, that's what we want, I just think it deserves a mention here.

We don't want `DefaultAzureCredential` to have configuration for this behavior though. It should always remember the first successful credential and use that exclusively.

Reply:
Oh sorry! I got that wrong. I’ll undo the change 👍